core/src/main/kotlin/org/evomaster/core/output/TestCaseWriter.kt

```diff
@@ -506,6 +509,12 @@ class TestCaseWriter {
                                 }
                             }
                         }
+                        else{
+                            // the object is empty
+                            if(format.isKotlin())  lines.add(".body(\"isEmpty()\", `is`(true))")
+                            else lines.add(".body(\"isEmpty()\", is(true))")
+                            //lines.add(".body(\"isEmpty()\", is(true))")
```

Comment:
is this commented // out line still necessary?

Reply:
Nope. Cleaned up the code a bit of other leftover comments.